modules/collection/misc/gtk.nix

```
@@ -0,0 +1,114 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}: let
+  inherit (lib.options) mkOption mkEnableOption;
+  inherit (lib.types) listOf package lines;
+  inherit (lib.modules) mkIf;
+  inherit (lib.rum.generators.gtk) toGtk2Text toGtkINI;
+  inherit (lib.strings) hasPrefix;
+  inherit (lib.lists) optionals any;
+  inherit (lib.types) attrsOf oneOf bool int float str;
+  inherit (builtins) attrNames;
```

```suggestion
  inherit (lib.attrsets) attrNames;
```
I would bring this in line with the other imports for consistency.

---

If a function has a `builtins` option, that's what I prefer to use as its the origin of the function.